longevity_in_memory_test.py

```diff
@@ -0,0 +1,17 @@
+from longevity_test import LongevityTest
+
+
+class InMemoryLongevetyTest(LongevityTest):
+
+    """
+    :avocado: enable
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(InMemoryLongevetyTest, self).__init__(*args, **kwargs)
+
+    def test_in_mem_longevity(self):
+        self._pre_create_schema()
```

Comment:
I decided to implement this way since I wanted to avoid changes in `_pre_create_schema` and `create_cf` methods (to avoid introducing potential bugs and since in-memory is special for enterprise and is not needed to be in general classes)

Reply:
I prefer you'll add it as a param to pre_create_schema and use it instead of altering after it was populated.
I think users most likely use it like that.